extensions/lorem-ipsum/src/utils.tsx

```diff
@@ -15,27 +18,63 @@ const generator = new LoremIpsum({
 // generator.generateWords(1);
 // generator.generateParagraphs(7);
 
-export const generateParagraph = () => {
-  return generator.generateSentences(10);
+export const generateParagraphs = (count: number) => {
+  return Array.from(Array(count))
+    .map(() => generator.generateSentences(6))
```

Comment:
This hardcoded 6 makes the parargraphs less dynamic imo. You think we could have this be a random number between say 4 and 10?

Reply:
It was set as default/_hard-coded_ to 10 before via `return generator.generateSentences(10);` 😕 😛 

Again, simple enough fix - go ahead and tweak to randomize if desired.

Reply:
Who will do the tweaks, and yes, it was hardcoded before as mentioned.

Are we going for something like `Math.floor(Math.random() * 7) + 4;`?

Reply:
Seems reasonable from my PoV... does not need to be _truly_ random 🤓 

Or perhaps re-use the options for LoremIpsum lib?

```js
const loremIpsumOptions = {
  sentencesPerParagraph: {
    max: 8,
    min: 4,
  },
  wordsPerSentence: {
    max: 16,
    min: 4,
  },
};

const generator = new LoremIpsum(loremIpsumOptions);
```

...

```js
export const generateParagraphs = (count: number) => {
  return Array.from(Array(count))
    .map(() =>
      generator.generateSentences(
        Math.floor(
          Math.random() *
            (loremIpsumOptions.sentencesPerParagraph.max - loremIpsumOptions.sentencesPerParagraph.min + 1)
        ) + loremIpsumOptions.sentencesPerParagraph.min
      )
    )
    .join("\r\n\r\n"); // newline + seperator line
};
```

Reply:
If it was hardcoded before, I'm fine with just keeping it as is! It's been a while since I wrote that, I don't remember all the details. Looks great!

Reply:
If we have that already added in the code it might be fine just reusing that @GastroGeek, can you do that?

Reply:
Well, that is my change to make that so its accessible but sure, will push up some tweaks in relation to this convo

Reply:
Done! @pernielsentikaer and updated CHANGELOG too...